src/main/java/io/appium/java_client/remote/IOSMobileCapabilityType.java

```diff
@@ -25,8 +25,8 @@
  * https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md#ios-only</a>
  * <br>
  * and<br>
- * <a href="https://github.com/appium/appium-xcuitest-driver#desired-capabilities">
- * https://github.com/appium/appium-xcuitest-driver#desired-capabilities</a>
+ * <a href="https://github.com/appium/appium-xcuitest-driver/blob/master/docs/capabilities.md">
+ * https://github.com/appium/appium-xcuitest-driver/blob/master/docs/capabilities.md</a>
```

Comment:
does it make sense to use website with the documentation: https://appium.github.io/appium-xcuitest-driver/ ?

Reply:
I think `https://appium.github.io/appium-xcuitest-driver/latest/capabilities/` would be the most accurate link